app/scripts/styles/theme.ts

```diff
@@ -16,7 +16,7 @@ export const VEDA_OVERRIDE_THEME = {
     popover: 1500,
     skipLink: 1600,
     toast: 1700,
-    tooltip: 1800
+    tooltip: 9999
```

Comment:
I edited to show the tooltip on top of Modal, and this probably is not ideal if there is a tooltip (not from the modal) and the modal at the same time on the screen. Do you know any better way to handle this? @danielfdsilva